CHANGELOG.md

```diff
@@ -1,29 +1,30 @@
 # Changelog
 
-## 1.2.0 (Unreleased)
+## 1.2.0
 
 ### Improvements
-* Now uses [OpenSSL 1.1.1.d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz)
+* Now uses [OpenSSL 1.1.1.d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz). [PR #60](https://github.com/corretto/amazon-corretto-crypto-provider/pull/60)
```

Comment:
Updated to ?

Reply:
I think that it is important to emphasize "uses". It could be "updated to use" if you think it important.

Reply:
Current one is fine.